public/data/shafaq-here.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Shafaq Israil",
+  "type": "personal",
+  "bio": "Open Source Enthusiast! | Learning Web Development | DevOps ",
```

There's an extra space here. I recommend removing it. What do you think?
```suggestion
  "bio": "Open Source Enthusiast! | Learning Web Development | DevOps",
```

---

> There's an extra space here. I recommend removing it. What do you think?
> ```suggestion
>   "bio": "Open Source Enthusiast! | Learning Web Development | DevOps",
> ```
> 

Hey, thanks for reviewing it and yeah I think I removing that space would be great.